pkgs/by-name/od/odoo16/package.nix

```diff
@@ -24,9 +24,11 @@ python.pkgs.buildPythonApplication rec {
   src = fetchzip {
     url = "https://nightly.odoo.com/${odoo_version}/nightly/src/odoo_${version}.zip";
     name = "odoo-${version}";
-    hash = "sha256-ICe5UOy+Ga81fE66SnIhRz3+JEEbGfoz7ag53mkG4UM="; # odoo
+    hash = "sha256-dBqRZ3cf4/udP9hm+u9zhuUCkH176uG2NPAy5sujyNc="; # odoo
   };
 
+  patches = [ ./fix-test.patch ];
```

Comment:
18.0 patch doesn't cleanly apply to 16.0, we need python311 (for fixing sphinx error) thus we need it.